google-cloud-storage/src/main/java/com/google/cloud/storage/multipartupload/model/ListMultipartUploadsResponse.java

```diff
@@ -0,0 +1,525 @@
+ */
+@BetaApi
+public final class ListMultipartUploadsResponse {
+
+  @JacksonXmlElementWrapper(useWrapping = false)
+  @JacksonXmlProperty(localName = "Upload")
+  private List<MultipartUpload> uploads;
+
+  @JacksonXmlProperty(localName = "Bucket")
+  private String bucket;
+
+  @JacksonXmlProperty(localName = "Delimiter")
+  private String delimiter;
+
+  @JacksonXmlProperty(localName = "EncodingType")
+  private String encodingType;
+
+  @JacksonXmlProperty(localName = "KeyMarker")
+  private String keyMarker;
+
+  @JacksonXmlProperty(localName = "UploadIdMarker")
+  private String uploadIdMarker;
+
+  @JacksonXmlProperty(localName = "NextKeyMarker")
+  private String nextKeyMarker;
+
+  @JacksonXmlProperty(localName = "NextUploadIdMarker")
+  private String nextUploadIdMarker;
+
+  @JacksonXmlProperty(localName = "MaxUploads")
+  private int maxUploads;
+
+  @JacksonXmlProperty(localName = "Prefix")
+  private String prefix;
+
+  @JsonAlias("truncated")
+  @JacksonXmlProperty(localName = "IsTruncated")
+  private boolean isTruncated;
+
+  @JacksonXmlElementWrapper(useWrapping = false)
+  @JacksonXmlProperty(localName = "CommonPrefixes")
+  private List<CommonPrefixHelper> commonPrefixes;
+
+  // Jackson requires a no-arg constructor
+  private ListMultipartUploadsResponse() {}
+
+  private ListMultipartUploadsResponse(
+      List<MultipartUpload> uploads,
+      String bucket,
+      String delimiter,
+      String encodingType,
+      String keyMarker,
+      String uploadIdMarker,
+      String nextKeyMarker,
+      String nextUploadIdMarker,
+      int maxUploads,
+      String prefix,
+      boolean isTruncated,
+      List<String> commonPrefixes) {
+    this.uploads = uploads;
+    this.bucket = bucket;
+    this.delimiter = delimiter;
+    this.encodingType = encodingType;
+    this.keyMarker = keyMarker;
+    this.uploadIdMarker = uploadIdMarker;
+    this.nextKeyMarker = nextKeyMarker;
+    this.nextUploadIdMarker = nextUploadIdMarker;
+    this.maxUploads = maxUploads;
+    this.prefix = prefix;
+    this.isTruncated = isTruncated;
+    if (commonPrefixes != null) {
+      this.commonPrefixes = new ArrayList<>();
+      for (String p : commonPrefixes) {
+        CommonPrefixHelper h = new CommonPrefixHelper();
+        h.prefix = p;
+        this.commonPrefixes.add(h);
+      }
+    }
+  }
+
+  /**
+   * The list of multipart uploads.
+   *
+   * @return The list of multipart uploads.
+   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  public ImmutableList<MultipartUpload> getUploads() {
+    return uploads == null ? ImmutableList.of() : ImmutableList.copyOf(uploads);
+  }
+
+  /**
+   * The bucket that contains the multipart uploads.
+   *
+   * @return The bucket name.
+   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  public String getBucket() {
```

Comment:
Similar to ListMultipartUploadsRequest can we change the getters and setters to be just resource value like bucket instead of getBucket and delimiter instead of setDelimiter.